### DISCUSSION TITLE
Buer-Nahida 我上午八

### DISCUSSION THREAD

**Discussion by VacuolePaoo** [share121/inter-knot#1515] [其他]
![容器 1](https://github.com/user-attachments/assets/548d986a-b74b-421d-aca4-9893ff342bfe)

**Comment by VacuolePaoo** [comment]
有能力访问空洞内网络的绳匠可以加入了，我们将会在频道中发布每个版本的更新内容

**Comment by niegong** [comment]
好耶😊😊

**Comment by bin-lin2** [comment]
这种情况能解决吗？
![微信图片_20240811233206](https://github.com/user-attachments/assets/ae02b081-85f2-4f3c-8a9d-dfedcdf1d9f6)

**Comment by capacitor1** [comment]
> Reply::这种情况能解决吗？

加载太慢导致的。实际上底下会有有一个人机验证。选自行车摩托车

**Reply by bin-lin2** [reply]
在后台挂了好久一点变动也没有，魔法换了几个节点都不管用

**Comment by 290678** [comment]
想到了谷歌似人的人机验证

**Comment by bin-lin2** [comment]
不知道discord用的什么，但我谷歌的验证能过

**Comment by sishui-buyin** [comment]
嗯...我进不去

**Comment by Yersmagit** [comment]
> 有能力访问空洞内网络的绳匠可以加入了，我们将会在频道中发布每个版本的更新内容

你说的最好是空洞内网络

**Reply by whatcanisaylaoda** [reply]
？？

**Comment by Inter-Knot-User** [comment]
好耶！

**Comment by TrafficLight6** [comment]
上不去可以试试tor，b站有教程

**Reply by TrafficLight6** [reply]
[这儿](https://www.bilibili.com/video/BV1Ri42197ig/?spm_id_from=333.1007.top_right_bar_window_custom_collection.content.click&vd_source=c5a9da41bff36395200ce94f69b0c9f6)

**Reply by Phaethon10031** [reply]
网站加载不出来，加速器也开了，是不是因为人太多了

**Comment by Phaethon10031** [comment]
网站加载不出来，加速器也开了，是不是因为人太多了

**Comment by uidsd604** [comment]
无法接受邀请是什么意思，满了吗

**Reply by HeroBrine-c** [reply]
没满

**Comment by Ben316-svg** [comment]
这你受得了吗

**Comment by HeroBrine-c** [comment]
![20b50895b8e06cdc0d24ac492e25401](https://github.com/user-attachments/assets/4ccc50f8-9b64-4848-ace7-77b448714ffa)
这.....

**Comment by capacitor1** [comment]
> Reply::这.....

啊？我这安装没一点问题 🤔

**Reply by HeroBrine-c** [reply]
我第一次安装都成功的，但号被封就删了，现在下回来又安装失败

**Reply by capacitor1** [reply]
> 我第一次安装都成功的，但号被封就删了，现在下回来又安装失败

我建议你根据Setup Log的内容判断到底为啥出错。

**Comment by cpu644** [comment]
@HeroBrine-c 你可以使用网页版discord 功能性一样 客户端就是套皮 
https://discord.com/channels/@me 自动跳转登录页的链接

**Reply by xxx-zzl** [reply]
好好好

**Comment by whatcanisaylaoda** [comment]
在Github上发干嘛？

**Comment by cpu644** [comment]
@VacuolePaoo 不是哥们那么大一个群呢

**Comment by AetherLude** [comment]
布什隔门

**Comment by AetherLude** [comment]
群呢？

**Reply by Buer-Nahida** [reply]
群被删了？

**Comment by Buer-Nahida** [comment]
不是哥们

**Comment by Buer-Nahida** [comment]
咋邀请链接没用了

**Reply by AetherLude** [reply]
> 咋邀请链接没用了

哥们你被和谐了

**Reply by Buer-Nahida** [reply]
> > 咋邀请链接没用了
> 
> 哥们你被和谐了

换小号也没用啊

**Comment by AetherLude** [comment]
群被解散了？

**Comment by Myitian** [comment]
消失了？

**Comment by Buer-Nahida** [comment]
不至于吧，ProBot移除不就行了？
